app/Tickets/Flipside/class.FlipsideTicketRequestEmail.php

```diff
@@ -41,7 +41,7 @@ public function getHTMLBody()
                 Reminder:  The dates for mailing your ticket request envelope are '.$this->settings['mail_start_date'].' through '
                 .$this->settings['request_stop_date'].'. Your postmark must be within that date range.<br><br>
                 If you need to update your ticket registration before mailing in your ticket request, return to 
-                <a href="https://secure.burningflipside.com/tickets/">secure.burningflipside.com/tickets/</a>.<br><br>
+                <a href="'.$this->secureUrl.'/tickets/">secure.burningflipside.com/tickets/</a>.<br><br>
```

Comment:
i see no unaddressed comments on any other PR.. 
i would expect your comments on this topic to be here.. https://github.com/BurningFlipside/CommonCode/pull/29

i want to do this right.. but i also want to focus on one task at a time.. 
pausing to refactor an existing implementation every time i draw your attention so some code isn't up to par..  is not in my opinion the right way.. 

if it isn't currently broken i want to do it later..

my primary goal is to complete the dev sandbox project.. then i'll be happy to circle back and address outstanding tickets on these repos..